Several modifications

amazzini opened the pull request:
1) TapiOdu: Removed + created again odu-mep augment of mep, was wrongly
on CEP
2) TapiPhotonic: remove the PowerPropertiesPac class, replacing it with
the PowerProperties data type (modification is backward compatible)
(OtsiTerminationPac, MediaChannelPropertiesPac,
OtsiTerminationConfigPac)
3) TapiPathComp: corrected comment
4) TapiConnectivity: Added recursive relationship to ConnectivityService
(ConnServHasSubordinateConnServ), new class ResiliencyRouteConstraint
5) removed resource/serviceSpec, replaced with GlobalClass
(Resource-spec was used by SIP, Link, Node, Topology, NEP,
InterRuleGroup, NodeRuleGroup, CEP, SwitchControl,  Connection,
Notification, MEG, Path)
(Service-spec was used by NetworkTopologyService, ConnectivityService,
NotifSubscrService, OamService, VirtualNtwService,
PathComputationService)

Commit:
Several modifications

1) TapiOdu: Removed + created again odu-mep augment of mep, was wrongly
on CEP
2) TapiPhotonic: remove the PowerPropertiesPac class, replacing it with
the PowerProperties data type (modification is backward compatible)
(OtsiTerminationPac, MediaChannelPropertiesPac,
OtsiTerminationConfigPac)
3) TapiPathComp: corrected comment
4) TapiConnectivity: Added recursive relationship to ConnectivityService
(ConnServHasSubordinateConnServ), new class ResiliencyRouteConstraint
5) removed resource/serviceSpec, replaced with GlobalClass
(Resource-spec was used by SIP, Link, Node, Topology, NEP,
InterRuleGroup, NodeRuleGroup, CEP, SwitchControl,  Connection,
Notification, MEG, Path)
(Service-spec was used by NetworkTopologyService, ConnectivityService,
NotifSubscrService, OamService, VirtualNtwService,
PathComputationService)

Commit:
Added comment in Eqp diagram

Commit:
read/write of odu-csep-ttp-pac/configured-client-type

Now "config true" as it should be provisionable (through
OduConnectivityServiceEndPointSpec)
Also small change of TapiCommon diagram (shown the
SipAugmentsLogRecordBody of TapiStreaming)

karthik-sethuraman reviewed (changes requested):
Could you commit the corresponding yang files (.tree files). Else it is not possible to review the changes

Commit:
YANG modules and other minor corrections

- TapiConnectivity, added missing "strictComposite" stereotypes to the
associations of new class ResiliencyRouteConstraints.
- TapiConnectivity, added missing inheritance from localClass to new
class ResiliencyRouteConstraints.

Commit:
Other minor corrections

ResilienceConstraint, removed “s” from _resiliencyRouteConstraints (now
aligned to Edge)

Commit:
tapiEquipment: supportingPhySpan augments link, not node

Other minor corrections
